website/plugins.js

```diff
@@ -1,10 +1,9 @@
 module.exports = [
   {
-    name: 'AdMob',
-    pub: 'google_mobile_ads',
+    name: 'AdMob (Deprecated)',
+    pub: 'firebase_admob',
     firebase: 'admob',
-    remoteSource: 'https://github.com/googleads/googleads-mobile-flutter',
-    documentation: 'https://pub.dev/documentation/google_mobile_ads/latest/',
+    documentation: '',
```

Comment:
Shouldn't this link to    ` documentation:'https://firebase.flutter.dev/docs/analytics/overview'`, ?

Reply:
An empty string renders no link on the UI like it was previously:
<img width="706" alt="Screenshot 2021-03-25 at 16 52 22" src="https://user-images.githubusercontent.com/16018629/112511809-ae0ca680-8d8a-11eb-81d7-913b5c94ed4e.png">